geoapps/iso_surfaces/constants.py

```diff
@@ -86,7 +86,7 @@
         "fixed_contours": {
             "main": True,
             "label": "Fixed Contours",
-            "value": 0.0,
+            "value": "",
```

Comment:
**question:** is it expecting at runtime a string with coma separated values?

Reply:
yes, but input_file converts the string to None if empty.